src/components/homePage/History.vue

```diff
@@ -0,0 +1,74 @@
+<template>
```

Comment:
Is there a reason About.vue wasn't used? That file was already created and implemented.

Reply:
That's my bad, I'll use that instead :)